packages/core/patternlab-config.json

```diff
@@ -75,6 +75,10 @@
   "patternExportPatternPartials": [],
   "patternExportPreserveDirectoryStructure": true,
   "patternExportRaw": false,
+  "patternTranslations": {
```

Comment:
I'm not quite sure if this is a good approach but giving the user the ability to change the automated generated texts seems to be a good idea.